ideascube/serveradmin/catalog.py

```diff
@@ -415,14 +439,22 @@ def __init__(self):
         self._cache_base_dir = settings.CATALOG_CACHE_BASE_DIR
         os.makedirs(self._cache_base_dir, exist_ok=True)
 
-        self._cache_remote_dir = os.path.join(self._cache_base_dir, 'remotes')
-        self._load_remotes()
-
-        self._cache_catalog = os.path.join(self._cache_base_dir, 'catalog.yml')
+        self._catalog_cache = os.path.join(self._cache_base_dir, 'catalog.yml')
         self._local_package_cache = os.path.join(
             self._cache_base_dir, 'packages')
+        os.makedirs(self._local_package_cache, exist_ok=True)
+
+        self._storage_base_dir = settings.CATALOG_STORAGE_BASE_DIR
+        os.makedirs(self._storage_base_dir, exist_ok=True)
```

Comment:
> Maybe we can call that one self._root?

I think the distinction between "cache" and "storage" is very important to keep.

A few months down the road, we'll have completely forgotten which is which.